mypy_primer/projects.py

```diff
@@ -893,6 +893,15 @@ def get_projects() -> list[Project]:
             pip_cmd="{pip} install types-setuptools types-psutil",
             name_override="CPython (peg_generator)",
         ),
+        Project(
+            location="https://github.com/python-trio/trio",
+            mypy_cmd="{mypy} src/trio",
+            pyright_cmd="{pyright} src/trio/_tests/type_tests",
```

Comment:
I took Eric's comment to suggest that it's valuable to run over everything and let the diff reveal any changes
```suggestion
            pyright_cmd="{pyright} src/trio",
```

When I ran, it looks like about half of the errors are maybe due to a difference in interpretation of type ignore in trio/socket.py. The pyright errors around the `GLOBAL_RUN_CONTEXT` look a little suspicious to me, not sure why it thinks that is `trio._core._run`